.github/workflows/ci.yml

```diff
@@ -73,6 +79,14 @@ jobs:
         with:
           maven-version: 3.8.6
 
+      - name: "Determine if dependency convergence should be skipped"
+        run: |
+          skip_dependency_convergence=${{ inputs.skip_dependency_convergence }}
+          if [ "skip_dependency_convergence" = "true" ]; then
```

Comment:
I would invert this whole logic.

`skip_dependency_convergence` (default: false) -> `run_dependency_convergence` (default: true), since it makes it easier to use in conditions.

This block can then be simplified to:
```
name: "Enable dependency convergence check"
if: ${{ inputs.run_dependency_convergence }}
run: echo "MVN_DEPENDENCY_CONVERGENCE=-Dflink.convergence.phase=install -Pcheck-convergence" >> $GITHUB_ENV
```

Then remove `MVN_DEPENDENCY_CONVERGENCE` from jobs.compile_and_test.env.